include/wx/generic/listctrl.h

```diff
@@ -130,6 +130,7 @@ class WXDLLIMPEXP_CORE wxGenericListCtrl: public wxNavigationEnabled<wxListCtrlB
     bool EndEditLabel(bool cancel);
 
     wxTextCtrl* GetEditControl() const;
+    bool IsVisible(long visible);
```

Comment:
```suggestion
    bool IsVisible(long item);
```
Should also get `const wxOVERRIDE`, see final comment.